src/bp/core/services/dialog/flow/service.ts

```diff
@@ -47,9 +48,62 @@ export class MutexError extends Error {
   type = MutexError.name
 }
 
+class FlowCache {
+  private _flows: Map<string, Map<string, FlowView>>
+
+  constructor() {
+    this._flows = new Map()
+  }
```

Comment:
The constructor is not really necessary, you can declare the new map when declaring your variable